database/migrations/2017_05_15_184344_add_soft_delete_to_tagging_tagged_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddSoftDeleteToTaggingTaggedTable extends Migration
```

Comment:
ah ok! I'll get rid of that. For some reason, I read [this](https://dosomething.slack.com/?redir=%2Farchives%2FC0YGXUE01%2Fp1494864214528044) as adding soft deletes, I'm not sure why!